.gitpod.yml

```diff
@@ -13,16 +13,13 @@ ports:
 
 tasks:
   - name: Forem Server
-    before: |
-      redis-server &
-    init: >
-      gitpod-init.sh
+    before: redis-server &
+    init: ./gitpod-init.sh
+    command: bin/startup
+  - name: Open Site
     command: >
-      bin/startup
-  - command: >
       gp await-port 3000 &&
-      printf "Waiting for Forem server ..." &&
-      until $(curl -sNL $(gp url 3000) | grep -q "Forem"); do printf '.'; sleep 5; done && echo "" &&
+      printf "Waiting for Forem instance to load in the browser..." &&
```

Comment:
After debugging a bit I realized the curl command would never succeed as the Gitpod URL requires the credentials of the currently logged in user. I've simplified this so the preview window opens once port 3000 is available. It will be blank initially, but as soon as the Rails server is up and running, it loads the local dev Forem instance.

Reply:
I tried testing this - I seem to still get the foreman not installed issue?

```
== Command ["foreman start -f Procfile.dev"] failed ==
gitpod /workspace/forem $ which foreman
gitpod /workspace/forem $ 
```

Reply:
Hmm, weird. Let me investigate.

Reply:
I'm seeing it now too. Not sure what's different as my last build on Gitpod was working. 🤔

```
🤙 This task ran as a workspace prebuild
🎉 Well done on saving 6 minutes

       _.-'                                           
              `-.__.-'                                               

903:M 17 Sep 2021 20:12:37.579 # Server initialized
903:M 17 Sep 2021 20:12:37.579 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
903:M 17 Sep 2021 20:12:37.580 * DB loaded from disk: 0.000 seconds
903:M 17 Sep 2021 20:12:37.580 * Ready to accept connections
== STARTING UP ==

== Command ["foreman start -f Procfile.dev"] failed ==
```

Reply:
Looks like the solution was to modify the `GEM_HOME` @djuber.